servicemesh-install/topics/install.adoc

```diff
@@ -27,301 +75,566 @@ The 3scale Istio Adapter is deployed and configured in the custom resource file.
-|`developer`
+|`memory`
+|This is the amount of memory that is requested in the environment
+|Allocatable memory in megabytes or gigabytes based on your environment's configuration
+|`1G`
 
-|github
-|`username`
-|Should be modified to reflect the  https://help.github.com/articles/signing-up-for-a-new-github-account/[GitHub account] you want to use to run the Fabric8 launcher.
-|N/A
+|Limits
+|`cpu`
+|The maximum number of CPUs that telemetry is permitted to use
+|A valid number of allocatable CPUs based on your environment's configuration
+|`4800m`
 
 |
-|`token`
-|GitHub https://github.com/settings/tokens[personal access token] you want to use to run the Fabric8 launcher.
-|N/A
+|`memory`
+|The maximum amount of memory that telemetry is permitted to use
+|Allocatable memory in megabytes or gigabytes based on your environment's configuration
+|`4G`
+|===
 
-|catalog
-|`filter`
-|Filter to apply to the Red Hat booster catalog.
-|`booster.mission.metadata.istio`
 
-|
-|`branch`
-|Version of the Red Hat booster catalog that should be used with Fabric8.
-|`v85`
+=== Istio Pilot example
 
-|
-|`repo`
-|GitHub repository to use for Red Hat booster catalog.
-|`https://github.com/fabric8-launcher/launcher-booster-catalog.git`
+[source,yaml]
+----
+  pilot:
+    resources:
+      requests:
+        cpu: 100m
+     autoscaleEnabled: false
+     traceSampling: 100.0
+----
+
+.Istio Pilot parameters
+|===
+|Parameter |Description |Values |Default
+
+|`cpu`
+|The number of CPUs requested in the environment
+|A valid number of allocatable CPUs based on your environment's configuration
+|`500m`
+
+|`memory`
+|The amount of memory requested in the environment
+|Allocatable memory in megabytes or gigabytes based on your environment's configuration
+|`2048Mi`
+
+|`traceSampling`
+|This value controls how often random sampling occurs. Note: increase for development or testing.
+|A valid number
+|`1.0`
 |===
 
-.3scale parameters
+=== Tracing
+
+.Tracing parameters
 |===
-|Parameter |Description |Values | Default
+|Parameter |Description |Value |Default
 
 |`enabled`
-|Whether to install the 3scale adapter
+|This enables tracing in the environment
 |`true`/`false`
-|`false`
+|`true`
+|===
+
+=== Kiali example
+
+[NOTE]
+====
+Kiali supports Oath authentication and dashboard users. By default, Kiali uses OpenShift Oauth, but you can enable a dashboard user by adding a dashboard user and passphrase.
+====
+
+[source,yaml]
+----
+  kiali:
+     enabled: true
+     hub: kiali/
+     tag: v0.16.2
+     #dashboard:
+       #user: admin
+       #passphrase: admin
+----
 
-|`prefix`
-|a prefix to apply to the 3scale adapter image name used in docker pull.
-|valid image repo
-|`quay.io/3scale/` if deployment_type is origin and `openshift-istio-tech-preview/` if deployment_type is openshift
+.Kiali parameters
+|===
+|Parameter |Description |Values |Default
+
+|`enabled`
+|This enables or disables Kiali in {ProductShortName}
```

Comment:
re: Rob's comment on https://issues.jboss.org/browse/MAISTRA-366.  I think we should probably revise this to read   "Kiali is installed by default.  If you do not want to install Kiali, change the value to 'enabled: false' to disable the Kiali installation.

Reply:
I'm inclined to agree. I'll get the language updated for the next PR update.

Reply:
This known issue is in the update I pushed to this PR.